.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "mounts": [
+    "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind"
+  ],
+  "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/vscode-extension,type=bind",
+  "workspaceFolder": "/workspaces/vscode-extension",
+  "postCreateCommand": "mkdir -p /workspaces/vscode-extension/bin && cp /bin/$(ls /bin/docker-language-server*) /workspaces/vscode-extension/bin && npm ci",
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "eslint.validate": [
+          "javascript",
+          "javascriptreact",
```

Comment:
```suggestion
```
I think this can be deleted as we do not anticipate adding any React code in the immediate future.